modules/pipebackend/pipebackend.cc

```diff
@@ -52,10 +52,14 @@ void CoWrapper::launch()
    if(d_cp)
       return;
 
+   if(d_command.size() == 0)
```

Comment:
note for next time: d_command.empty() is a better habit. Does not actually matter here, but on some containers it is a lot faster.